service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisDocumentGeneratorService.java

```diff
@@ -59,4 +62,15 @@ public DocmosisDocument generateDocmosisDocument(Map<String, Object> templateDat
 
         return new DocmosisDocument(docmosisTemplate.getDocumentTitle(), response);
     }
+
+    public String generateDraftWatermarkEncodedString() {
+        InputStream is = getClass().getResourceAsStream("/assets/images/draft-watermark.png");
+        byte[] fileContent = new byte[0];
+        try {
+            fileContent = is.readAllBytes();
+        } catch (IOException e) {
+            log.error("Unable to generate draft water image for template.", e);
+        }
+        return Base64.getEncoder().encodeToString(fileContent);
```

Comment:
[-1] I think this already is implemented in DocmosisTemplateDataGeneration?

Reply:
Mistake from the massive merge conflict that I had to deal with probably